helmfile/charts/calico-felix-metrics/templates/service.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Don't we need anything else than adding this? Is the metrics enabled by default?

Reply:
Just saw that this is enabled via kubespray :+1: